make dist for auth fails on master

I just wanted to build auth for master for Debian. I used the new rules from build-scripts/README.md
- autoreconf -i
- ./configure --without-modules --without-dynmodules --disable-depedency-tracking
- make dist

the last command fails:

```
make[2]: Entering directory `/home/darilion/software/pdns4-fuer-neue-pakete/github-master-for-DS-bug-demonstration2/docs'
make[2]: *** No rule to make target `pdns_server.1', needed by `distdir'.  Stop.

```

--- a/docs/Makefile.am
+++ b/docs/Makefile.am
@@ -78,7 +78,7 @@ upload-docs: all-docs
 	rsync -crv --no-p --chmod=g=rwX --exclude '*~' ./PowerDNS-Authoritative.pdf web1.powerdns.com:/srv/www/doc.powerdns.com/authoritative/
 
 else # if HAVE_VIRTUALENV
-$(dist_MANS):
+$(MANPAGES_DIST):
 	echo "You need virtualenv to generate the manpages"
 	exit 1
 

